src/style.css

```diff
@@ -0,0 +1,94 @@
+* {
```

Comment:
do not overuse `*`-selector, style the elements itself,
if you want to reset body margins, just style body tag